src/components/TypeTester/_type-tester.scss

```diff
@@ -39,6 +39,12 @@
     'Bitstream Vera Sans Mono', Courier, monospace;
 }
 
+.#{$prefix}--mobile-column {
+  @include carbon--breakpoint("sm") {
+    margin-top: 0 !important;
```

Comment:
Nice! Looks good, just one comment: we usually avoid using `!important`. It seems like it works fine without it.

Reply:
Ohh gotcha - I'll update - sorry for the delay! @jnm2377